src/renderer/VideoButton.tsx

```diff
@@ -79,7 +88,7 @@ export default function VideoButton(props: any) {
   /**
   * Delete a video.
   */
-  function deleteVideo(filePath: string) {6
```

Comment:
as if I checked this into main... :| 
this is why I need to write some unit tests

Reply:
I actually thought that _I_ had made a mistake and couldn't for the life of me figure out where, but accepted that it just needed to be fixed regardless. 🤣

Reply:
hilariously it somehow isn't broken... no idea why

Reply:
Just javascript things, really. Javascript will take what you write and do the best it can, mostly ignoring things that doesn't make sense. 🤷‍♂️🤣